lib/safe_active_record/load.rb

```diff
@@ -15,8 +15,46 @@
 # limitations under the License.
 
 require 'safe_active_record/safe_query_manager'
+require 'set'
 
 module SafeActiveRecord
+  @visited = Set.new
+  @gemlist = Set.new
+
+  def self.init_visited
+    if $LOADED_FEATURES.is_a? Enumerable
+      @visited.merge($LOADED_FEATURES)
+      # Also merge the relative imported paths
+      $LOADED_FEATURES.each do |path|
+        $LOAD_PATH.each{ |start_path| @visited.add(path[start_path.size+1..-4]) if path.start_with?(start_path)}
```

Comment:
Can you add a comment to explain the offset (`size+1..-4`) for future reference?

Reply:
Done :)